Makefile

```diff
@@ -296,7 +284,7 @@ swagger: reports
 	fi; \
 	(set -o pipefail && py.test $(PYTEST_ARGS) awx/conf/tests/functional awx/main/tests/functional/api awx/main/tests/docs --release=$(VERSION_TARGET) | tee reports/$@.report)
 
-check: flake8 pep8 # pyflakes pylint
+check: black
```

Comment:
I'm opposed to making it mandatory.

Reply:
Can you clarify this a bit?  Making what mandatory?